### ISSUE TITLE
Breaks with PIP v10?

### ISSUE THREAD

**Issue by djpeacher** [marteinn/Skeppa#11]
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-8HV8oz/skeppa/setup.py", line 9, in <module>
        from pip.req import parse_requirements
    ImportError: No module named req
```

Suggested change:
```
try: # for pip >= 10
    from pip._internal.req import parse_requirements
except ImportError: # for pip <= 9.0.3
    from pip.req import parse_requirements
```

**Comment by marteinn** [comment]
Thanks for the issue @djpeacher! I solved this issue by inlining the deps (96d7ebb473e00e8e306fa881f60e3c53bbc6e648). 

A new version with this fix is out now: https://github.com/marteinn/Skeppa/releases/tag/v2.0.2

**Comment by djpeacher** [comment]
Thank you @marteinn 👍
